http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -941,12 +1031,18 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         .asScala
         .toList
     )
+
+    // TODO: this currently matches on the 'name' (actually type) of database so for example
```

Comment:
Names are unique within a workspace, and there can only be 1 shared app of a given type per workspace, so this works. Agree about thinking about future options though.

Also would be good to move away from using WSM `enumerateResources` in favor of a `getByName` API or something. That has implications on auth domains I think (cc @aherbst-broad )